tests/packagedcode/data/pypi/pipfile.lock/sample2/Pipfile.lock

```diff
@@ -0,0 +1,184 @@
+{
+    "_meta": {
+        "hash": {
+            "sha256": "6e45251662433bf51f96fb3d2204b65416fece329d60e6235c0f0edc416cfe24"
```

Comment:
Does this value coorespond to the actual `Pipfile.lock`'s sha256 value?

If so, we can probably collect this info on the Package's `sha256` field.

Reply:
@majurg Do we actually need to collect this because Pipfile.lock is the dependency files and I think collecting dependency will be enough. What do you think??
If we have to collect please inform me :)

Reply:
Yeah, this probably doesn't need to be on the top-level package, as it is just a lockfile.